spec/definitions.yaml

```diff
@@ -942,6 +942,7 @@ definitions:
     properties:
       aws:
         type: object
+        x-nullable: true
```

Comment:
@marians Could this introduce problems for existing clients/customers? On AWS I'd expect the value to always be there and on Azure there are no existing clients yet. Can I just merge this or do I need to do any additional work here?

Reply:
Changing a request body spec in this way makes the API more tolerant and should not cause any problems to clients.

Reply:
Merging should only happen after the change specified here is implemented and the API is rolled out, so that the api-spec `master` always represents what users can do with the API.

Reply:
Ack. Thanks! :+1: